src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -39,6 +39,8 @@ FixedwingPositionControl *l1_control::g_control;
 static int _control_task = -1;			///< task handle for sensor task */
 
 FixedwingPositionControl::FixedwingPositionControl() :
+	_sub_airspeed(ORB_ID(airspeed), 0, 0, nullptr),
```

Comment:
If I'm not wrong, this airspeed topic is generated from raw sensor data? How does this tie in with the EKF-estimated true airspeed, and why would we use the raw value? Just trying to get a better idea of how FW controllers work.